### PR TITLE
LVRUBY-55: Add Sidekiq for background job

### PR DESCRIPTION
**Pull request by olenaniemova**
## JIRA ticket

[LVRUBY-55](http://ssu-jira.softserveinc.com/browse/LVRUBY-55)

## Code reviewers

- [x] @kmandziy 
- [ ] @okohuch 

## Summary of issue

Send mail to another stream

## Summary of change

Add Sidekiq and devise_async for background job
Add config for Redis
Add config for sidekiq and smtp

## Review checklist

- [x] Rubocop doesn't raise any warnings and offenses
- [x] New features are covered with new specs
- [x] All specs pass

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.0%** covered at **1.78** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

File: config/environments/development.rb [hunk 1]
```diff
@@ -32,12 +32,30 @@
...
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.perform_deliveries = true
+
+  config.action_mailer.raise_delivery_errors = true
+
   config.action_mailer.delivery_method = :letter_opener
+  # config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = {
```
Comment (okohuch) [inline config/environments/development.rb]: remove the code instead of commenting out if you do not need it

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.6%** covered at **2.74** hits/line

**Pull request merged**